client/error.go

```diff
@@ -160,3 +171,27 @@ func statusFromError(err error) (*status.Status, bool) {
 	}
 	return nil, false
 }
+
+var apiDisabledRe = regexp.MustCompile("^(.*) has not been used.*or it is disabled")
+
+// IsApiDisabledError tests if the given error is about a disabled API. If so, returns the name of the disabled API
+// extracted from error and true value. Otherwise (the error is not about disabled API), returns empty string and false.
```

Comment:
We need to be hitting the API that determines if the API is enabled rather than just parsing the error for a couple of reasons:
1. This will reduce the total number of calls CQ has to make as we won't need to make calls that will fail
2. Users have complained about the fact that CQ pollutes their log security tools because every call we make to a disabled service produces a log record